### PR TITLE
net pricing: update prices

### PR DESCRIPTION
**Pull request by Ksenia-Galtsova**

### CODE REVIEW

File: channel-manager-api.md [hunk 1]
```diff
@@ -34,27 +34,50 @@ Mews automatically sends changes in Inventory \(once connection is set up\). Dat
...
-               "guestCount": 2,
-               "amount": 100,
-               "currencyCode": "USD"
-            }
+   {
+    "spaceTypeCode": "D1",
+    "ratePlanCode": "FF",
+    "prices": [
+        {
+         ~~"amount": 100.00,~~
```
Comment (patrikmiksovsky95) [inline channel-manager-api.md]: As discusssed ~~ should be removed and used only in table part

Comment (Ksenia-Galtsova) [reply channel-manager-api.md]: done

**Comment by siroky** [comment]
@vitasamek the PR is so broken, that it broke even Github 😂

**Comment by patrikmiksovsky95** [comment]
https://github.com/MewsSystems/gitbook-channel-manager-api/pull/45

**Pull request merged**